### PR TITLE
add last updated to product page template

### PR DESCRIPTION
**Pull request by pglevy**
I tried this thing. What do you think?

### CODE REVIEW

**Comment by kbchumley** [comment]
looks good

**Review by pglevy — COMMENTED** [review]
Is there an auto-date we can use instead of creating that content for every page?

**Review by kbchumley — APPROVED** [review]
this works - is there an auto date we can use?

File: _layouts/product.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     {% include breadcrumbs.html %}
 
     <h1 class="margin-top-0">{{ page.title }}</h1>
-
+      <small>Last updated: {{ page.date }}</small>
```
Comment (pglevy) [inline _layouts/product.html:14]: Why are you using small tag here?

**Review by bnaolu — APPROVED** [review]
Nice!

**Pull request merged**